packages/now-cli/package.json

```diff
@@ -15,6 +15,7 @@
     "test-unit": "nyc ava test/*unit.js --serial --fail-fast --verbose",
     "test-integration": "ava test/integration.js --serial --fail-fast",
     "test-integration-now-dev": "ava test/dev/integration.js --serial --fail-fast --verbose",
+    "test-integration-now-build": "ava test/build/integration.js --serial --fail-fast --verbose",
```

Comment:
I don't believe these new tests are running in CI.

You will need to configure Circle CI.

Reply:
Added it to CircleCi